scripts/Muon/GUI/Common/contexts/frequency_domain_analysis_context.py

```diff
@@ -70,3 +72,7 @@ def get_names_of_time_domain_workspaces_to_fit(
     @property
     def window_title(self):
         return "Frequency Domain Analysis"
+
+    @staticmethod
+    def data_type_options_for_sequential():
+        return ["All", "FD_Re", "FD_Im", "FD_mod", "MaxEnt"]
```

Comment:
should it be FD_MaxEnt for consistency? Or do you mean FFT_Re, to signify the method?

Reply:
`FD_Re` and the others are the strings seen at the end of the workspace names when you finish a FFT or MaxEnt calculation. I would need to change the output workspace names from these calculations if we want to change these data type options

Reply:
its ok, was just a question